### ISSUE TITLE
jcarousel have support dynamic add or remove an item?

### ISSUE THREAD

**Issue by sjpeng** [jsor/jcarousel#817]
Currently, my implementation is using empty after append jcarousel.
jcarousel have support dynamic add or remove an item?

**Comment by jsor** [comment]
Yes, you can add/remove items. Be sure to call [`reload`](http://sorgalla.com/jcarousel/docs/reference/api.html#reload) afterwards.
